packages/cactus-common/src/main/typescript/key-converter.ts

```diff
@@ -0,0 +1,128 @@
+import KeyEncoder from "key-encoder";
+
+export enum KeyFormat {
+  Raw = "raw",
+  Hex = "hex",
+  PEM = "pem",
+}
+
+export class KeyConverter {
+  private keyEncoder: KeyEncoder;
+
+  public readonly supportedKeyFormats: string[];
+
+  constructor() {
+    this.supportedKeyFormats = Object.values(KeyFormat);
+    this.keyEncoder = new KeyEncoder("secp256k1");
+  }
+
+  /**
+   * Convert public key from one format to another format
+   * @param key
+   * @param fromFormat
+   * @param toFormat
+   */
+  public publicKeyAs(
+    key: string | Uint8Array,
```

Comment:
I might be wrong, but Typescript has standard API to handle cryptographic key data like, CryptoKey. https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.cryptokey.html
For nodejs, they has another set of API for handling keys, can you use them instead ? https://nodejs.org/api/crypto.html#crypto_crypto_createpublickey_key
I thought we better to have a functionality to check whether key is valid or not.